example/app/config.yml

```diff
@@ -519,11 +519,35 @@ jk2:
   stackNameNoDash: jk2
 
 mboyd-test:
```

Comment:
Won't everyone need a prefix now?

Reply:
Yes, which I can add for everyone. Just didn't bother before getting a review of whether others are okay with this change